proto/insonmnia.go

```diff
@@ -54,25 +53,19 @@ func (m *EthAddress) Unwrap() common.Address {
 	return common.BytesToAddress(m.Address)
 }
 
-func (m *EthAddress) MarshalYAML() (interface{}, error) {
-	if m != nil {
-		return m.Unwrap().Hex(), nil
-	}
-	return nil, nil
+func (m EthAddress) MarshalText() ([]byte, error) {
+	return []byte(m.Unwrap().Hex()), nil
 }
 
-func (m *EthAddress) UnmarshalYAML(unmarshal func(interface{}) error) error {
-	var v string
-	if err := unmarshal(&v); err != nil {
-		return err
-	}
-
+func (m *EthAddress) UnmarshalText(text []byte) error {
+	v := string(text)
 	if !common.IsHexAddress(v) {
-		return errors.New("invalid ethereum address format")
+		return fmt.Errorf("invalid ethereum address format \"%s\"", v)
```

Comment:
What do you expect here to see? What if it's not a string, but unprintable byte array?

Reply:
It used just for notifications of human (when parsing YAMLs). 
If an error message will include the wrong value it will be easier to detect where's it comes from.